File: app/src/main/java/com/kotobyte/models/Origin.java
package com.kotobyte.models;

import android.support.annotation.Nullable;

public class Origin {

    private String mLanguage;
    private String mText;

    public Origin(String language, String text) {
        mLanguage = language;
        mText = text;
    }

    public String getLanguage() {
        return mLanguage;
    }

    @Nullable
    public String getText() {
        return mText;
    }
}

File: app/src/main/java/com/kotobyte/models/Origin.kt
package com.kotobyte.models

import android.os.Parcel
import android.os.Parcelable

data class Origin(
        val language: String,
        val text: String?

) : Parcelable {

    override fun writeToParcel(parcel: Parcel, flags: Int) {
        parcel.writeString(language)
        parcel.writeString(text)
    }

    override fun describeContents(): Int = 0

    companion object CREATOR : Parcelable.Creator<Origin> {

        override fun createFromParcel(parcel: Parcel): Origin =
                Origin(parcel.readString(), parcel.readString())

        override fun newArray(size: Int): Array<Origin?> = arrayOfNulls(size)
    }

}
